libs/components/help-inline/testing/src/help-inline/help-inline-harness.ts

```diff
@@ -0,0 +1,85 @@
+import { HarnessPredicate } from '@angular/cdk/testing';
+import { TemplateRef } from '@angular/core';
+import { SkyComponentHarness } from '@skyux/core/testing';
+import {
+  SkyPopoverContentHarness,
+  SkyPopoverHarness,
+} from '@skyux/popovers/testing';
+
+import { SkyHelpInlineHarnessFilters } from './help-inline-harness.filters';
+
+export class SkyHelpInlineHarness extends SkyComponentHarness {
+  /**
+   * @internal
+   */
+  public static hostSelector = 'sky-help-inline';
+
+  #getInlineHelpButton = this.locatorFor('.sky-help-inline');
+
+  /**
+   * Gets a `HarnessPredicate` that can be used to search for a
+   * `SkyHelpInlineHarness` that meets certain criteria
+   */
+  public static with(
+    filters: SkyHelpInlineHarnessFilters,
+  ): HarnessPredicate<SkyHelpInlineHarness> {
+    return SkyHelpInlineHarness.getDataSkyIdPredicate(filters);
+  }
+
+  /**
+   * Clicks the help inline icon button
+   */
+  public async click(): Promise<void> {
+    await (await this.#getInlineHelpButton()).click();
+  }
+
+  public async getAriaControls(): Promise<string | null> {
```

Comment:
Based on the feedback for the `help-inline.component.ts`, we may not need methods for `getAriaControls`, `getAriaExpanded`, `getAriaLabel` since those would be internal workings and not something the consumer should worry about.